### PR TITLE
Updated to work with multiple `scheduled` tags prefixed with the environment

### PR DESCRIPTION
**Pull request by willmcpo**
Made changes to make it work with multiple environments.

### CODE REVIEW

File: src/completeFlagDeployment.js [hunk 1]
```diff
@@ -5,19 +5,27 @@ import Logger from './log';
...
+    operations.push({
       op: 'replace',
       path: '/description',
       value: description,
-    }
-  ]);
+    });
+  }
+
+  const body = JSON.stringify(operations);
```
Comment (yusinto) [inline src/completeFlagDeployment.js]: Nit: extraneous newline here

Comment (yusinto) [inline src/completeFlagDeployment.js]: Nit: Extraneous newline here

File: src/completeFlagDeployment.test.js [hunk 2]
```diff
@@ -7,18 +7,7 @@ import getRequestHeaders from './getRequestHeaders';
...
-      path: '/tags',
-      value: ['non-scheduled'],
-    },
-    {
-      op: 'replace',
-      path: '/description',
-      value: 'this is a test flag',
-    }
-  ]);
+
```
Comment (yusinto) [inline src/completeFlagDeployment.test.js]: Nit again: extraneous newline

File: src/scheduler.js [hunk 3]
```diff
@@ -54,10 +55,10 @@ export default async({environment, apiKey, slack}) => {
 
       const currentDateTime = moment();
       const targetDeploymentDateTime = moment(outstandingTask.targetDeploymentDateTime, 'YYYY-MM-DD HH:mm Z');
-      const isLegible = currentDateTime.isAfter(targetDeploymentDateTime);
+      const isScheduledTimePassed = currentDateTime.isAfter(targetDeploymentDateTime);
 
-      // log.info(`Found scheduled flag ${f.key} with targetDeploymentDateTime: ${targetDeploymentDateTime}. IsLegible? ${isLegible}`);
-      return isLegible;
+      log.info(`Found scheduled flag ${f.key} with targetDeploymentDateTime: ${targetDeploymentDateTime}. isScheduledTimePassed? ${isScheduledTimePassed}`);
```
Comment (yusinto) [inline src/scheduler.js]: This logging string will output ${targetDeploymentDateTime} in unix epoch elapsed format which is probably not what you want. Do you mean to format it in a more readable format?

File: README.md [hunk 4]
```diff
@@ -44,7 +44,8 @@ By default, ld-scheduler polls launch darkly every 60 seconds:
     });
     ```
 
-3. Tag your feature flag. In launch darkly's dashboard, under the Settings tab of your feature flag, under Tags, add a tag called "scheduled". 
+3. Tag your feature flag. In launch darkly's dashboard, under the Settings tab of your feature flag, under Tags, add one or more "scheduled" tags, each prefixed with an environment name.
```
Comment (yusinto) [inline README.md:47]: I have updated a new screenshot to reflect the tag changes. The new image link is [here](https://cloud.githubusercontent.com/assets/1593077/23781221/ddd4557e-059f-11e7-8e4f-601e6cf91f78.png)

**Review by yusinto — CHANGES_REQUESTED** [review]
Thanks for the PR! I have a few comments which are minor, you could take a look that would be awesome. Thanks again!

**Comment by willmcpo** [comment]
@yusinto ok changes made and prepublished. Can you review again? Thanks.

**Review by yusinto — APPROVED** [review]
Awesome stuff! Thanks @willmcpo !

**Pull request merged**